test/requests.js

```diff
@@ -661,11 +645,6 @@ const tests = function(web3) {
       assert(receipt.contractAddress, "should have deployed a contract");
     });
 
-    it("should return null for the to field due to contract creation (eth_getTransactionReceipt)", async function() {
```

Comment:
Same as above ^ this may have been removed by accident from PR [212](https://github.com/trufflesuite/ganache-core/pull/212)?

Reply:
This may be some squashing magic gone wrong